bemoji

```diff
@@ -209,7 +213,8 @@ case "$exit_value" in
         bm_cmds+=(_clipper)
     fi
     for cmd in "${bm_cmds[@]}"; do
-        echo "$result" | "$cmd"
+        [ "$bm_echo_newline" = true ] && echo_opts= || echo_opts=-n
```

Comment:
Do we have to overwrite echo_opts to be empty here? Is there a path where it could already be set somewhere else?

Reply:
Done!

Reply:
Meh, I hit **Comment** at the wrong conversation 🤦

I meant: No, we don't define `echo_opts` anywhere else. It could be defined in the user's environment though 😎

Anyway, I think it's better to be explicit than implicit.